templates/source/sourceTemplate.md

```diff
@@ -0,0 +1,35 @@
+# Event Source for **\_\_\_\_**
+
+This event target receives .......
+
+## Prerequisites
+
+- unordered list out prerequisites
+- unordered list out prerequisites
+- unordered list out prerequisites
+
+##### This should be about a prerequisites
```

Comment:
We go straight from h2 to h4, this is not great. If the problem is the styling applied by Mkdocs, we should fix the styling, not hack the h levels.

Reply:
Oh sorry, I will override the styles. I didn't know that those "#" were the equivelent to h2 and h5. I do know how to make HTML semantic and use the correct ones for SEO. I had never really used markdown and didn't know what those #'s were for. Thanks for the review :)